pkgs/by-name/sy/sysvol/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gtk4-layer-shell,
+  gtkmm4,
+  pkg-config,
+  nix-update-script,
+  wireplumber,
+  wrapGAppsHook4,
+}:
+
+stdenv.mkDerivation {
+  pname = "sysvol";
+  version = "0-unstable-2024-06-07";
+
+  src = fetchFromGitHub {
+    owner = "System64fumo";
+    repo = "sysvol";
+    rev = "56d7dcda4b246e71b2c6d29cbb2315bddf446032";
+    hash = "sha256-WOcy2R0El1vl57Zimb7Hoh9XYTnH/zJS1n+gcaTU4V8=";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace-fail 'pkg-config' ''${PKG_CONFIG}
+  '';
```

Comment:
this is needed when cross compiling (where we can end up with multiple pkg-config's in play -- one for the build machine and one for the host machine -- and the unflavored `pkg-config` isn't put on PATH), e.g. `pkgsCross.aarch64-multiplatform.sysvol`. unfortunately a bit difficult to reproduce atm since one of sysvol's dependencies doesn't cross compile on master -- you'd have to cherry-pick [this](https://github.com/NixOS/nixpkgs/pull/306407) to build it.